### PR TITLE
PornHub: Fix video page

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

File: websites/P/PornHub/presence.ts [hunk 1]
```diff
@@ -15,51 +15,57 @@ function getTimestamps(
...
+        ),
+        presenceData: PresenceData = {
+          details: title ? title.innerText : "Title not found...",
+          state: uploader ? uploader.textContent : "Uploader not found...",
+          largeImageKey: "lg",
+          smallImageKey: video.paused ? "pause" : "play",
+          smallImageText: video.paused
+            ? (await strings).pause
+            : (await strings).play,
+          startTimestamp: timestamps[0],
```
Comment (ririxidev) [inline websites/P/PornHub/presence.ts:50]: don't need to set startTimestamp

File: websites/P/PornHub/presence.ts [hunk 2]
```diff
@@ -15,51 +15,57 @@ function getTimestamps(
...
-        state: uploader ? uploader.textContent : "Uploader not found...",
-        largeImageKey: "lg",
-        smallImageKey: video.paused ? "pause" : "play",
-        smallImageText: video.paused
-          ? (await strings).pause
-          : (await strings).play,
-        startTimestamp: timestamps[0],
-        endTimestamp: timestamps[1]
-      };
+        timestamps = getTimestamps(
```
Comment (ririxidev) [inline websites/P/PornHub/presence.ts:38]: use function from `Presence` class and remove function in this code.

**Review by ririxidev — CHANGES_REQUESTED** [review]
upload screenshots too

File: websites/P/PornHub/presence.ts [hunk 3]
```diff
@@ -15,51 +15,57 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  const startTime = Date.now();
+  const endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
   return [Math.floor(startTime / 1000), endTime];
 }
```
Comment (Bas950) [inline websites/P/PornHub/presence.ts:21]: use presence.getTimestamps instead

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by TheGamerzs** [comment]
image
image
image

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**